app/assets/javascripts/utilities/showUserAlertModal.js

```diff
@@ -128,18 +138,18 @@ function buildModalDiv(title, text, confirm_text) {
 
 /**
  * Takes template HTML for a modal and creates a DOM node based on supplied arguments
- * 
+ *
  * @private
  * @function getModal
- * 
+ *
  * @param {string} title The title/heading text to be displayed
  * @param {string} text The body text to be displayed
  * @param {string} confirm_text Text of the confirmation button
- * 
+ *
  * @returns {Element} DOM node of alert modal with formatted text
  */
 function getModal(title, text, confirm_text) {
   let wrapper = document.createElement('div');
-  wrapper.innerHTML= getModalHtml(title, text, confirm_text);
+  wrapper.innerHTML = getModalHtml(title, text, confirm_text);
```

Comment:
Oh nice, what linter are you using for Javascript?

Reply:
🤦‍♀️  I will fix. And I'm taking JS linting suggestions ...

Reply:
Done!